cli/src/main/kotlin/OrtMain.kt

```diff
@@ -208,11 +208,11 @@ class OrtMain : CliktCommand(name = ORT_NAME, invokeWithoutSubcommand = true) {
 
         """
             ________ _____________________
-            \_____  \\______   \__    ___/ the OSS Review Toolkit, version $version.
-             /   |   \|       _/ |    |
-            /    |    \    |   \ |    |    Running$command as '$user' under Java ${env.javaVersion} on ${env.os}
-            \_______  /____|_  / |____|    with ${env.processors} CPUs and a maximum of $maxMemInMib MiB of memory.
-                    \/       \/
+            \    _  \\______   \__    ___/ The OSS Review Toolkit, version $version.
+            |    |   \|       _/ |    |
+            |    |    \    |   \ |    |    Running$command as '$user' under Java ${env.javaVersion} on ${env.os}
+            \_________/____|___/ |____|    with ${env.processors} CPUs and a maximum of $maxMemInMib MiB of memory.
+
```

Comment:
We could drop this blank line now (or the `postfix = "\n"` below, but I'd prefer to drop the blank line).

Reply:
I tried without the blank line, but didn't like that the environment variable list below is not separated:
```
________ _____________________
\    _  \\______   \__    ___/ the OSS Review Toolkit, version d704115481-dirty.
|    |   \|       _/ |    |
|    |    \    |   \ |    |    Running as 'user' under Java 17.0.5 on Linux
\_________/____|___/ |____|    with 8 CPUs and a maximum of 7936 MiB of memory.
Environment variables:
ORT_CONFIG_DIR = ~/.ort/config
ORT_DATA_DIR = ~/.ort
SHELL = /bin/bash
TERM = xterm-256color
```
```
________ _____________________
\    _  \\______   \__    ___/ the OSS Review Toolkit, version d704115481-dirty.
|    |   \|       _/ |    |
|    |    \    |   \ |    |    Running as 'user' under Java 17.0.5 on Linux
\_________/____|___/ |____|    with 8 CPUs and a maximum of 7936 MiB of memory.

Environment variables:
ORT_CONFIG_DIR = ~/.ort/config
ORT_DATA_DIR = ~/.ort
SHELL = /bin/bash
TERM = xterm-256color
```
Actually looks worse on the console because there is less spacing between the lines.